guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -0,0 +1,62 @@
+---
+title: View models
+contributor_name: Space 48
+contributor_link: https://www.space48.com/
+
+---
+
+A view model is an abstraction of the view exposing public properties and commands. It allows users to offload features and business logic from block classes into separate classes that are easier to maintain, test, and reuse.
```

Comment:
Add a blank line after this paragraph. Also, consider changing "It allows users" to "It allows developers" or similar noun.